sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CTESubstitution.scala

```diff
@@ -130,23 +169,36 @@ object CTESubstitution extends Rule[LogicalPlan] {
    * @param plan the plan to be traversed
    * @return the plan where CTE substitution is applied
```

Comment:
As the return value of `traverseAndSubstituteCTE` changed a bit, this docs could use some update.